titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/KubeNotificationProcessor.java

```diff
@@ -264,6 +266,16 @@ private Task attachNodeMetadata(Task task, V1Node node) {
 
         acceptNotNull(node.getMetadata().getName(), nodeName -> agentAttributes.put(TaskAttributes.TASK_ATTRIBUTES_KUBE_NODE_NAME, nodeName));
 
+        List<V1NodeAddress> addresses = node.getStatus().getAddresses();
```

Comment:
The first ipv4 ip address should be used as the host ip like in:
https://github.com/Netflix/titus-control-plane/blob/master/titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/KubeApiServerIntegrator.java#L453

Reply:
@corindwyer I have seen your recent PR. We should share your address filter in both places. Once you merge, I will rebase and create a function in `KubeUtil`.

Reply:
Sounds good. I already merged.